db/schema.rb

```diff
@@ -11,23 +11,13 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-<<<<<<< HEAD
-ActiveRecord::Schema.define(version: 20161206021535) do
```

Comment:
@ryanveroniwooff I think this one is newer, looks like the date is 1206, the other one is 1205. Let's keep this and delete the older one.